lib/diaspora_federation.rb

```diff
@@ -14,6 +14,63 @@
 require "diaspora_federation/salmon"
 
 # diaspora* federation library
+#
+# In order to communicate with the application which uses the diaspora_federation gem
+# callbacks are introduced. The callbacks are used for getting required data from the
+# application or posting data to the application.
+#
+# Callbacks are implemented at the application side and must follow these specifications:
+#
+# fetch_person_for_webfinger
+#   Fetches person data from the application in a WebFinger document format
```

Comment:
it's data for the WebFinger, not "WebFinger document format" (which is xml/json), same is for  HCard, it's for the HCard, but not jet HCard format.